src/main/java/de/tracetronic/jenkins/plugins/ecutest/report/atx/ATXReportUploader.java

```diff
@@ -663,8 +664,11 @@ public TestInfoHolder call() throws IOException {
              * @return the result set
              * @throws SQLException in case of a SQL exception
              */
+            @SuppressFBWarnings(value = "SQL_PREPARED_STATEMENT_GENERATED_FROM_NONCONSTANT_STRING",
+                                justification = "Constant query statements used only")
             public ResultSet query(final String sql) throws SQLException {
-                return statement.executeQuery(sql);
+                statement = connection.prepareStatement(sql);
```

Comment:
```
                try (Statement statement = connection.createStatement()) {
                    statement.execute(sql);
                    return statement.getResultSet();
                }
```

Reply:
Closing Statement also closes returning ResultSet which is reused later.